sys/include/net/gnrc/ipv6/util.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup net_gnrc_ipv6_util     GNRC IPv6 utility functions
+ * @ingroup  net_gnrc_ipv6
+ * @brief   GNRC IPv6 utility functions
+ * @{
+ *
+ * @file
+ * @brief   GNRC IPv6 utility functions
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef NET_GNRC_IPV6_UTIL_H
+#define NET_GNRC_IPV6_UTIL_H
+
+#include "net/ipv6/addr.h"
+#include "net/gnrc/netif.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Configures a prefix delegation lease that is provided by the server.
+ *
+ * @param[in] netif     Network interface the prefix delegation was for.
+ * @param[in] pfx       Prefix for the prefix delegation.
+ * @param[in] pfx_len   Length of @p pfx in bits.
+ * @param[in] valid     Valid lifetime of the prefix delegation.
+ * @param[in] pref      Preferred lifetime of the prefix delegation.
+ * @param[in] tentative Add new prefix as tentative.
+ */
+void gnrc_util_conf_prefix(gnrc_netif_t *netif, const ipv6_addr_t *pfx,
```

Comment:
This might surprise you, but why not move it to `gnrc_netif_conf_prefix()` in `net/gnrc/netif/internal.h` or maybe `net/gnrc/netif/helper.h`? Since in this case this is all about interface configuration (setting a prefix in the NIB for an interface, setting an address for that prefix on an interface, ...), I think in this case it fits there.